dbt/models/iasworld/schema/iasworld.dweldat.yml

```
@@ -851,20 +869,63 @@ sources:
                   meta:
                     description: sfla (Building Square Footage) should not be null
               - accepted_range:
-                  name: iasworld_dweldat_sfla_sf_between_1_and_10000
-                  min_value: 1
-                  max_value: 10000
-                  additional_select_columns: *select-columns
+                  name: iasworld_dweldat_sfla_between_1_and_10000
+                  min_value: |
+                    CASE
+                      WHEN class = '202' THEN 1
+                      WHEN class = '203' THEN 1000
+                      WHEN class = '204' THEN 1801
+                      WHEN class = '205' THEN 1
+                      WHEN class = '206' THEN 2201
+                      WHEN class = '207' THEN 1
+                      WHEN class = '208' THEN 3801
+                      WHEN class = '209' THEN 5000
+                      WHEN class = '210' THEN 1
+                      WHEN class = '211' THEN 1
+                      WHEN class = '212' THEN 1
+                      WHEN class = '234' THEN 1
+                      WHEN class = '278' THEN 2001
+                      WHEN class = '295' THEN 1
+                      ELSE 1
+                    END
+                  max_value: |
+                    CASE
+                      WHEN class = '202' THEN 999
+                      WHEN class = '203' THEN 1800
+                      WHEN class = '204' THEN 10000
+                      WHEN class = '205' THEN 2200
+                      WHEN class = '206' THEN 4999
+                      WHEN class = '207' THEN 2000
+                      WHEN class = '208' THEN 4999
+                      WHEN class = '209' THEN 20000
```

You'd be surprised how big some of these places get:

```suggestion
                      WHEN class = '209' THEN 50000
```